app/Http/Controllers/HomeController.php

```diff
@@ -2,11 +2,23 @@
 
 namespace App\Http\Controllers;
 
-use Illuminate\Http\Request;
+use App\Models\Celeb;
+use App\Models\Movie;
+use App;
 
 class HomeController extends Controller
 {
     public function index(){
-        return view('home');
+        $movies = Movie::with(['reviews', 'genres'])
+            ->take(4)
+            ->get()
+            ->map(function ($movie) {
+                $movie->average_rating = round($movie->reviews->average('rating'),1);
+                return $movie;
+            })
+            ->sortByDesc('average_rating');
+        $celebs= Celeb::take(4)
+                        ->get();
```

Comment:
To make this easier to read, you can give add an empty line between segments and before the return. 

This is the format you should follow in all of your code. Have you set up the PhpStorm to use the format I have suggested, this makes it easier ;D. 

Also, when you passing multiple values to the view, you do not need separate arrays. You can do it like the example below. 

```
public function index()
    {
        $movies = Movie::with(['reviews', 'genres'])
            ->take(4)
            ->get()
            ->map(function ($movie) {
                $movie->average_rating = round($movie->reviews->average('rating'), 1);
                return $movie;
            })
            ->sortByDesc('average_rating');

        $celebs = Celeb::take(4)
            ->get();

        return view('home', [
            'movies' => $movies,
            'celebs' => $celebs
        ]);
    }
```

Reply:
Also too, you do not need the comments before the methods, as the method names and dependencies are pretty self-explanatory. So you can remove it :D